runtime/experimental/helpers/export.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+cp /tmp/php-$(php -r 'echo phpversion();').zip /export/php-fpm-$(php -r 'echo phpversion();').zip
+cp /tmp/php-cli-$(php -r 'echo phpversion();').zip /export/php-default-$(php -r 'echo phpversion();').zip
```

Comment:
Is it possible to simplify here (and elsewhere) by not having the version name in the temporary files/folders?

Reply:
It is.

I was envisioning a scenario where the runtime system and default layers are managed internally to bref. That is, we would build and publish all the PHP Version layers that we support out of the box, to all regions we support out of the box. In that scenario, it would be very rare for someone outside of bref to build/publish.

If we keep the version in the names, we can look in the S3 bucket(s) to see a history of the various versions we have uploaded. Ultimately, I envision having some associated meta-data with the packages in S3 that designate the `layer:version` that package is associated with.

In that scenario, it seemed good to automate building all the php packages and then uploading/registering them.

Which sorta brings up the question of how we afford to pay for S3 hosting. I will open an issue to chase that rabbit though. 

Ultimately, we can certainly simplify it, it will just add further complication if we desire to automate the building of multiple versions. It also complicates tracking the php packages, by version, in an S3 bucket. But certainly doable.

Reply:
I see, let's revisit that later when things settle down ^^